### ISSUE TITLE
offline icon

### ISSUE THREAD

**Issue by nik2208** [ionic-team/ionicons#599]
an offline icon (same as wifi but slashed) would be useful to indicate actions related to missing of network/connection

**Comment by adriangrahl** [comment]
I'm looking for this same kind of icon

**Comment by CaitlinWeb** [comment]
Surprising that this isn't already done, since there are "off" versions of other icons like mic. It would be a very useful addition.

**Comment by tbetous** [comment]
I agree with that ! There is nice off logo for volume, notification, micro and so on but not for offline. Actually I think a off logo for wifi would be a great addition !

**Comment by EuanDScott** [comment]
Even me! I find the Ionic icons very useful, especially when you are limited on time and money. This would be really good to have as the last 2 apps that I have worked on all need to have offline functionality.

I really want to do some graphical way of letting the user know that they are offline rather than my current alert approach.

**Comment by ViktorTyultinovBC** [comment]
It has been quite a while and there is still no icon for this...

**Comment by joachim23344** [comment]
I also need this icon

**Comment by christian-bromann** [comment]
Consolidating into #1449
